### PR TITLE
updates open api conventions to align snippets on kiota latest

### PR DESCRIPTION
**Pull request by baywet**
This PR reflects the updates in kiota done for open api conventions in that PR https://github.com/microsoft/kiota/pull/573

### CODE REVIEW

File: CodeSnippetsReflection.OpenAPI/KiotaOpenApiSchemaExtensions.cs [hunk 1]
```diff
@@ -6,7 +6,7 @@
 // THIS CLASS IS COPIED FROM KIOTA TO GET THE SAME NAMEING CONVENTIONS, WE SHOULD FIND A WAY TO MUTUALIZE THE CODE
```
Comment (irvinesunday) [inline CodeSnippetsReflection.OpenAPI/KiotaOpenApiSchemaExtensions.cs:6]: [Nitpick] *NAMEING* typo

**Comment by irvinesunday** [comment]
Should we add tests to this class?

**Comment by irvinesunday** [comment]
Add license info.

File: CodeSnippetsReflection.OpenAPI/KiotaOpenApiSchemaExtensions.cs (hunk 1 above)
Comment (baywet) [reply CodeSnippetsReflection.OpenAPI/KiotaOpenApiSchemaExtensions.cs:6]:
```suggestion
// THIS CLASS IS COPIED FROM KIOTA TO GET THE SAME NAMING CONVENTIONS, WE SHOULD FIND A WAY TO MUTUALIZE THE CODE
```

**Comment by baywet** [comment]
@irvinesunday about the unit tests, this is already unit tested in kiota. I don't see value in duplicating the tests.
Hopefully at some point we can just take a dependency on kiota an avoid duplicating code, but so far the impact is limited IMHO.

**Review by irvinesunday — APPROVED** [review]

**Pull request merged**